### ISSUE TITLE
moduleForComponent accepts 3 or 1 argument, should also accept 2

### ISSUE THREAD

**Issue by dandehavilland** [switchfly/ember-test-helpers#52]
Works: `moduleForComponent('pie-chart');`
Works: `moduleForComponent('pie-chart', 'PieChartComponent', {});`

Errors: `moduleForComponent('pie-chart', 'PieChartComponent');`
```
'undefined' is not an object (evaluating 'callbacks.needs')
```

/cc @sbl

**Comment by rwjblue** [comment]
This has been resolved and released in https://github.com/switchfly/ember-test-helpers/pull/48 and released as ember-test-helpers@0.4.5.  It is also included in ember-qunit@0.3.3.

**Comment by rwjblue** [comment]
Whoops, wait, let me confirm that...

**Comment by dandehavilland** [comment]
I'm running ember-cli@0.2.5, that introduced the failures for me (upgraded from 0.2.3)

**Comment by rwjblue** [comment]
The ember-cli version is not directly related to the version of this library (though it does ship with a default).  What version of ember-qunit are you using?

**Comment by dandehavilland** [comment]
```
"ember-qunit": "0.3.3"
```

**Comment by dandehavilland** [comment]
Thanks for the quick response!

**Comment by rwjblue** [comment]
Released as v0.3.4 of ember-qunit (and 0.4.6 of ember-test-helpers).
